TAC.rst

```diff
@@ -0,0 +1,68 @@
+######################################
+Technical Advisory Council (RSTUF-TAC)
+######################################
+
+The role of the Technical Advisory Council (TAC) is to provide subject matter
+expertise on software repository design, to make recommendations and guide the
+future activities of the technical community.
+
+Get Involved
+############
+
+If you are maintaining a public repository and would like to join the
+discussion, just open a PR adding your details in the TAC Members table.
+TAC members will review it at the next meeting.
+
+Meetings
+########
+
+The TAC members meet (and vote if there is a quorum) on-demand as part of the
+`RSTUF Community Meeting <https://repository-service-tuf.readthedocs.io/en/stable/devel/contributing.html#meetings>`_.
+
+TAC Roles and Responsibilities
+##############################
+
+Voting Member
+=============
+
+* Shares use cases
+* Provides subject matter expertise on software repository design
+* Votes on user/repository features priority in the roadmap (each organization
+  has one vote)
+
+.. note::
+    Functional features, such as support, stability, securing, strength,
+    behavior, etc., have high-priority on user/repository features.
+
+RSTUF-TAC Members
+#################
+
+.. list-table:: RSTUF-TAC Members
+    :header-rows: 1
+    :widths: 20 20 40 20
+
+    * - Representative
+      - Role
+      - Organization
+      - Github
+    * - Kairo de Araujo
+      - Voting Member
+      - RSTUF Maintainer
+      - @kairoaraujo
+    * - Martin Vrachev
+      - Voting Member
+      - RSTUF Maintainer
+      - @MVrachev
```

Comment:
We need to add @lukpueh and @KAUTH  as well and make it clear that all of us as RSTUF maintainers can have 1 vote.
Also, let's make it clear that one can be a representative in multiple organizations, but can vote as a representative of one of them. That is logical, but is important to note. For example, even though @lukpueh is an RSTUF maintainer he can join as a voting member on behalf of TUF, but won't vote as an RSTUF maintainer.